workers/ohsome_quality_analyst/ohsome/layer_definitions.yaml

```diff
@@ -236,3 +236,22 @@ mapaction_rivers_length:
     Length of objects identified as rivers (or riverbanks)
   endpoint: elements/length
   filter: waterway in (riverbank,river) and type:way
+
+ideal_vgi_infrastructure:
+  name: IDEAL-VGI Infrastructure
+  description: Objects related to infrastructure
+  endpoint: elements/length
+  filter: (aerialway=* or aeroway=* or highway=* or power=* or railway=* or telecom=*) and geometry:line
+
+ideal_vgi_poi:
+  name: IDEAL-VGI Points-of-Interests
+  description: Features related to Points-of-Interests
+  endpoint: elements/count
+  filter: (amenity=* or craft=* or emergency=* or historic=* or leisure=* or man_made=* or office=* or place=* or shop=* or tourism=*) and geometry:point
```

Comment:
By limiting this type to points some elements will be missed which I don't see mentioned in the documentation. I would not include relations but for Heidelberg you miss about 10-20% of elements: https://api.ohsome.org/v1/elements/count/groupBy/type?bboxes=8.67%2C49.39%2C8.71%2C49.42&filter=amenity%3D*%20or%20craft%3D*%20or%20emergency%3D*%20or%20historic%3D*%20or%20leisure%3D*%20or%20man_made%3D*%20or%20office%3D*%20or%20place%3D*%20or%20shop%3D*%20or%20tourism%3D*&format=json&time=2021-08-01

```
{
  "attribution": {
    "url": "https://ohsome.org/copyrights",
    "text": "© OpenStreetMap contributors"
  },
  "apiVersion": "1.6.0",
  "groupByResult": [
    {
      "groupByObject": "NODE",
      "result": [
        {
          "timestamp": "2021-08-01T00:00:00Z",
          "value": 4862
        }
      ]
    },
    {
      "groupByObject": "WAY",
      "result": [
        {
          "timestamp": "2021-08-01T00:00:00Z",
          "value": 941
        }
      ]
    },
    {
      "groupByObject": "RELATION",
      "result": [
        {
          "timestamp": "2021-08-01T00:00:00Z",
          "value": 24
        }
      ]
    }
  ]
}
```

Reply:
+1 for not restricting to just nodes. I'd say that one could extend the request to polygons as well, i.e. `… and (geometry:point or geometry:polygon)`

Reply:
You're correct, it shouldn't only contain `geometry:point`. I would suggest `(geometry:point or geometry:polygon)`. I checked `geometry:line` as well, but this includes a lot of features that are imho no POIs like these:
https://www.openstreetmap.org/way/26088966
https://www.openstreetmap.org/way/158025770
https://www.openstreetmap.org/way/312561922
https://www.openstreetmap.org/way/196967009

I would suggest:
 * use these `point` and `polygon` geometries
 * remove `emergency` key, as it isn't a key defining the primary use of a feature, mostly used on roads
 * remove `place` key as it denotes settlements, not POIs
 * the `man_made` key includes a lot of non-POI features like bridges, masts, embankments, …
 * think how to deal with the `amenity` key as this includes a lot of non-POI features

To get an idea of the complexity of this issue you can have a look at the [POI display OSM Wiki](https://wiki.openstreetmap.org/wiki/POI_display) page.

Reply:
created #161 for further discussion